### ISSUE TITLE
trained model with adversarial resulting blank image

### ISSUE THREAD

**Issue by JamesLuoau** [alexjc/neural-enhance#34]
Hi, I have tried both with perceptual loss and adversarial, the perceptual model create a much blur images compare to "ne4x-small-0.1.pkl", the adversarial trained model resulting a blank image after about 25 epochs. 

my training data is just 15 images I took randomly in my office. 

please note I'm using the script you change with issue #31 

Thanks

**Comment by JamesLuoau** [comment]
Here is my log about training

python3 enhance.py --train ./office/*.JPG --model custom1 --scales=2 --epochs=20 --perceptual-layer=conv5_2 --smoothness-weight=2e4 --adversary-weight=1e3 --generator-start=5 --discriminator-start=0 --adversarial-start=5  --discriminator-size=64 --save-every=1 --device=gpu

Super Resolution for images and videos powered by Deep Learning!
  - Code licensed as AGPLv3, models under CC BY-NC-SA.
  - Using the device `gpu` for neural computation.

Training 20 epochs on random image sections with batch size 15.
  - 44 tensors learned for generator.
  - 21 tensors learned for discriminator.

Epoch #1 at 227.3s, lr=1.00e-05
  - generator total=1.61e+03 prcpt=1.61e+03 smthn=5.57e-03 advrs=0.00e+00
  - discriminator 0.860288 15 -0.860288 15
  - saving current generator layers to disk...
  - Saved model as `ne4x-custom1-0.1.pkl.bz2` after training.
Epoch #2 at 456.9s, lr=1.00e-04
  - generator total=1.54e+03 prcpt=1.54e+03 smthn=4.96e-03 advrs=0.00e+00
  - discriminator 0.988683 15 -0.988683 15
  - saving current generator layers to disk...
  - Saved model as `ne4x-custom1-0.1.pkl.bz2` after training.
Epoch #3 at 686.4s, lr=1.00e-04
  - generator total=1.47e+03 prcpt=1.47e+03 smthn=4.96e-03 advrs=0.00e+00
  - discriminator 0.997327 15 -0.997328 15
  - saving current generator layers to disk...
  - Saved model as `ne4x-custom1-0.1.pkl.bz2` after training.

could you see any thing wrong?

**Comment by alexjc** [comment]
You should start pre-training with 2_2 first. Does that work?

**Comment by JamesLuoau** [comment]
Thank you for your reply.

Yes 2_2 goes well. Just clarifying do you mean I have to train 2_2 first and using the same trained model as base to train 5_2? Sorry I'm very new in this area.

**Comment by JamesLuoau** [comment]
Thank you very much @alexjc, confirmed and should pre-training with 2_2. Will close this.

**Comment by elhoussinetalab** [comment]
@JamesLuoau what do you mean pre-training with 2_2? 

Thank you.

**Comment by rachitagrawal** [comment]
@elhoussinetalab you should use --perceptual-layer=conv2_2 instead of --perceptual-layer=conv5_2 in the above command line argument
